frontend/packages/core/src/AppLayout/notifications.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import styled from "@emotion/styled";
+import {
+  Box,
+  ClickAwayListener,
+  Grow,
+  IconButton,
+  MenuList,
+  Paper,
+  Popper,
+} from "@material-ui/core";
+import NotificationsIcon from "@material-ui/icons/Notifications";
+
+const StyledNotificationsIcon = styled(IconButton)`
+  color: #ffffff;
+  margin-right: 1rem;
+  border-radius: 8px;
+  padding: 8px;
+  &:hover {
+    background: #2d3db4;
+  }
+  &:active {
+    background: #2938a5;
+  }
+`;
+
+const Notifications: React.FC = () => {
+  const [open, setOpen] = React.useState(false);
+  const anchorRef = React.useRef(null);
+
+  const handleToggle = () => {
+    setOpen(!open);
+  };
+
+  const handleClose = event => {
+    if (anchorRef.current && anchorRef.current.contains(event.target)) {
+      return;
+    }
+    setOpen(false);
+  };
+
+  function handleListKeyDown(event) {
+    if (event.key === "Tab") {
+      event.preventDefault();
+      setOpen(false);
+    }
+  }
+
+  return (
+    <Box>
+      <StyledNotificationsIcon
+        ref={anchorRef}
+        edge="end"
+        aria-controls={open ? "notifcation-options" : undefined}
+        aria-haspopup="true"
+        onClick={handleToggle}
+      >
+        <NotificationsIcon />
+      </StyledNotificationsIcon>
+      <Popper open={open} anchorEl={anchorRef.current} role={undefined} transition>
+        {({ TransitionProps, placement }) => (
+          <Grow
+            {...TransitionProps}
+            style={{ transformOrigin: placement === "bottom" ? "center top" : "center bottom" }}
```

Comment:
Emotion supports [prop evaluation](https://emotion.sh/docs/styled#changing-based-on-props) that you could use here